SU-HSDO/stanford_mrc
# Mrcd8 153 events list & featured event view

# READY FOR REVIEW

# Summary
- Added mrc_date module to help with formatters for date fields
- Added mrc events view for events listing & featured event blocks
- preprocess the patterns to clean them up to make it easier in the templates

# Needed By (Date)
- 12/22

# Urgency
- low

# Steps to Test

1. checkout `MRCD8-153-event-list` branch of [mrc_blt](https://github.com/SU-HSDO/mrc_blt/pull/6)
2. `composer update --prefer-source`
3. checkout this branch of stanford_mrc
4. `drush updb -y`
5. create some events that occur in the future
6. place the events blocks on a page to see the view.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
